src/global/global.h

```diff
@@ -179,6 +179,7 @@ struct Parameters {
 #ifdef DE
   int out_float32_GasEnergy = 0;
 #endif
+  int output_always = 0;
```

Comment:
I might explicitly make this a bool rather than an int. Other than that looks great.